streams/src/test/java/org/apache/kafka/streams/state/internals/AbstractWindowBytesStoreTest.java

```diff
@@ -1176,6 +1105,7 @@ private void putFirstBatch(final WindowStore<Integer, String> store,
         store.put(0, "zero", startTime);
         store.put(1, "one", startTime + 1L);
         store.put(2, "two", startTime + 2L);
+        store.put(3, "three", startTime + 2L);
```

Comment:
add 2 records at the same timestamp to test the forward and backward fetch cases.